tools/clang_tools/api_booster/main.cc

```diff
@@ -102,24 +108,36 @@ class ApiBooster : public clang::ast_matchers::MatchFinder::MatchCallback,
   }
 
 private:
+  static bool isEnvoyNamespace(absl::string_view s) {
+    return absl::StartsWith(s, "envoy::") || absl::StartsWith(s, "::envoy::");
+  }
+
+  static std::string truncateForDebug(const std::string& text) {
+    const uint32_t MaxExpansionChars = 250;
+    return text.size() > MaxExpansionChars ? text.substr(0, MaxExpansionChars) + "..." : text;
+  }
+
   // Match callback for TypeLoc. These are explicit mentions of the type in the
   // source. If we have a match on type, we should track the corresponding .pb.h
   // and attempt to upgrade.
   void onTypeLocMatch(const clang::TypeLoc& type_loc, const clang::SourceManager& source_manager) {
+    absl::optional<clang::SourceRange> source_range;
     const std::string type_name =
         type_loc.getType().getCanonicalType().getUnqualifiedType().getAsString();
     // Remove qualifiers, e.g. const.
     const clang::UnqualTypeLoc unqual_type_loc = type_loc.getUnqualifiedLoc();
-
-    // Today we are only smart enough to rewrite ElaborateTypeLoc, which are
+    DEBUG_LOG(absl::StrCat("Type class ", type_loc.getType()->getTypeClassName()));
+    // Today we are only smart enought to rewrite ElaborateTypeLoc, which are
```

Comment:
typo "enought". Is the spell checker running correctly on these files? We seem to have had strange spell checking issues in here previously.

Reply:
The pedantic spell checker wasn't pointing at `./tools`. Fixed, very small change so I'll include it here.